server/src/error.rs

```diff
@@ -204,6 +207,7 @@ impl Display for EdgeError {
             EdgeError::NotReady => {
                 write!(f, "Edge is not ready to serve requests")
             }
+            EdgeError::InvalidTokenOnClosedMode => write!(f, "Edge is running in closed mode and the token is not subsumed by any registered tokens"),
```

Comment:
This needs to be updated to strict behavior I assume?

Reply:
Yes, nice catch! https://github.com/Unleash/unleash-edge/pull/474/commits/136373471997b062052369cabb8de68eb879b324